### PR TITLE
Add default platform versions

### LINKED ISSUE #38554: Add TPVs for every platform and their respective major .NET version.
### Type of issue

Missing information

### Description

It would be useful to know the default TPV used for each respective .NET version.

i.e. net6.0-android -> net6.0-androidYY.0

net7.0-android -> net7.0-androidYY.0

net8.0-android -> net8.0-androidYY.0

That way we know what the primary TPV would be used by default of those SDK releases.

We provide a bit of this information here: 

https://www.nuget.org/packages/Microsoft.Maui.Graphics#supportedframeworks-body-tab

But would be very useful on this documentation page for major platforms as a table of some sort.

### Page URL

https://learn.microsoft.com/en-us/dotnet/standard/frameworks

### Content source URL

https://github.com/dotnet/docs/blob/main/docs/standard/frameworks.md

### Document Version Independent Id

4352856b-5ccf-8e62-48b4-3151340cac56

### Article author

@gewarren

### Metadata

* ID: 49205137-dcec-a120-e5e7-1b9c79cdafbb 
* Product: **dotnet**
* Technology: **dotnet-standard**

---
[Associated WorkItem - 206396](https://dev.azure.com/msft-skilling/Content/_workitems/edit/206396)

### PR DESCRIPTION
**Pull request by gewarren**
Fixes #38554.

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/standard/frameworks.md](https://github.com/dotnet/docs/blob/ff5f1ba3fae17e35bd409fc0619d7351657f6940/docs/standard/frameworks.md) | [Target frameworks in SDK-style projects](https://review.learn.microsoft.com/en-us/dotnet/standard/frameworks?branch=pr-en-us-39582) |

### CODE REVIEW

File: docs/standard/frameworks.md [hunk 1]
```diff
@@ -73,6 +73,7 @@ The following table shows the compatibility of the .NET 5+ TFMs.
 | net7.0-ios         | (Subsequent version of `net6.0-ios`)                                                                                    |
 | net7.0-maccatalyst | (Subsequent version of `net6.0-maccatalyst`)                                                                            |
 | net7.0-macos       | (Subsequent version of `net6.0-macos`)                                                                                  |
+| net7.0-tizen       | TBD (plus everything else inherited from `net7.0`) |
```
Comment (gewarren) [inline docs/standard/frameworks.md:76]: Need some help on this one.

File: docs/standard/frameworks.md [hunk 2]
```diff
@@ -98,10 +100,35 @@ Use these guidelines to determine which TFM to use in your app:
...
+| .NET 8       | Mac Catalyst | 17.2        |
+| .NET 6       | macOS        | 12.0        |
+| .NET 7       | macOS        | 13.0        |
+| .NET 8       | macOS        | 14.2        |
+| .NET 6       | tvOS         | 15.1        |
+| .NET 7       | tvOS         | 16.1        |
+| .NET 8       | tvOS         | 17.1        |
+| .NET 7       | Tizen        | 7.0         |
+| .NET 8       | Tizen        | 8.0         |
+| .NET 6       | Windows      |             |
```
Comment (gewarren) [inline docs/standard/frameworks.md:128]: @JonDouglas Any idea what these Windows versions should be?

Comment (JonDouglas) [reply docs/standard/frameworks.md:128]: https://github.com/dotnet/sdk/blob/be20b697b83861d0e34ebe0341b65216c1186d0b/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Windows.targets#L32-L34

Should be 7.0 it looks like for all.

File: docs/standard/frameworks.md (hunk 1 above)
Comment (JonDouglas) [reply docs/standard/frameworks.md:76]: I think as is works. @terrajobst may have the most clarity here.

File: docs/standard/frameworks.md [hunk 3]
```diff
@@ -73,6 +73,7 @@ The following table shows the compatibility of the .NET 5+ TFMs.
 | net7.0-ios         | (Subsequent version of `net6.0-ios`)                                                                                    |
 | net7.0-maccatalyst | (Subsequent version of `net6.0-maccatalyst`)                                                                            |
 | net7.0-macos       | (Subsequent version of `net6.0-macos`)                                                                                  |
+| net7.0-tizen       | `tizen40` (plus everything else inherited from `net7.0`)            |
```
Comment (terrajobst) [inline docs/standard/frameworks.md:76]: Seems like you'll want to keep the same number of spaces for the table:

```suggestion
| net7.0-tizen       | `tizen40` (plus everything else inherited from `net7.0`)                                                                |
```

File: docs/standard/frameworks.md [hunk 4]
```diff
@@ -81,6 +82,7 @@ The following table shows the compatibility of the .NET 5+ TFMs.
 | net8.0-ios         | (Subsequent version of `net7.0-ios`)                                                                                    |
 | net8.0-maccatalyst | (Subsequent version of `net7.0-maccatalyst`)                                                                            |
 | net8.0-macos       | (Subsequent version of `net7.0-macos`)                                                                                  |
+| net8.0-tizen       | (Subsequent version of `net7.0-tizen`)                              |
```
Comment (terrajobst) [inline docs/standard/frameworks.md:85]: Same as above:
```suggestion
| net8.0-tizen       | (Subsequent version of `net7.0-tizen`)                                                                                  |
```

**Review by terrajobst — APPROVED** [review]

**Comment by terrajobst** [comment]
This makes sense, although I can't vouch for the correctness of the suggested TPV -- so I take those on face value :-)

**Pull request merged**